packages/api/emails/hipaa/html.pug

```diff
@@ -70,11 +73,12 @@ div.main
     | will not be extended.
   p
     | Lastly, please understand that this application process
-    | does not guarantee that the nominated individual/family will receive a grant from Keep Swimming Foundation.
-    | It is imperative that the
+    span.text__bold.text__italic  does not guarantee that the nominated individual/family will receive a grant from Keep Swimming Foundation.
```

Comment:
Is the text italic working correctly? From the screenshot, it doesn't appear to be italicized.

Reply:
<img width="489" alt="Screen Shot 2021-08-02 at 6 35 27 PM" src="https://user-images.githubusercontent.com/66797751/127936154-f981f62c-563e-4cc7-8459-47dc3c97cda1.png">
This is the most updated email sent it's got the italics